app/tests/spec/models/auth_brokers/fx-desktop.js

```diff
@@ -116,6 +116,7 @@ define([
         return broker._notifyRelierOfLogin(account)
           .then(function () {
             assert.equal(data.email, 'testuser@testuser.com');
+            assert.isFalse(data.verified);
```

Comment:
These three `assert.isFalse(data.verified)`s make me scratch my head. I realize it defaults to `false`, but why are we checking it here?